core/services/vrf/delegate_test.go

```diff
@@ -54,7 +54,7 @@ type vrfUniverse struct {
 	ks        keystore.Master
 	vrfkey    vrfkey.KeyV2
 	submitter common.Address
-	txm       *txmmocks.TxManager
+	txm       *txmmocks.TxManager[*evmtypes.Address, *evmtypes.TxHash, *evmtypes.BlockHash]
```

Comment:
could we wrap this as `*txmmocks.EvmTxManager`? seems like it's getting use a lot in tests

Reply:
Done. It was used in 4 places.
Note that I couldn't do it in txmmocks package, as files there are autogenerated.
I did it in evmtypes.utils.go

Reply:
Actually reverted it back. Import cycles if I try doing that :(

Reply:
> Note that I couldn't do it in txmmocks package, as files there are autogenerated.
technically, I think you can but it just won't be updated with it

might be possible to define it in a helpers_test.go file in the txmgr module?

Reply:
I could add to mock package, but don't want to, since when it gets updated and removed in future, it will confuse someone unnecessarily.

txmodule won't work, because txmmocks depends on txmmodule already.
I could create a separate testutil namespace separately, but wasn't convinced its worthwhile.

Reply:
I have created types in those files though.